sw/device/silicon_creator/lib/BUILD

```diff
@@ -277,9 +277,9 @@ cc_library(
     name = "shutdown",
     srcs = ["shutdown.c"],
     deps = [
-        ":shutdown_intf",
         ":error",
         ":log",
+        ":shutdown_intf",
```

Comment:
This reorder is due to the requirement these deps be sorted as enforced by buildifier